server/statistics/buckets/bucket_stat_informer.go

```diff
@@ -18,21 +18,19 @@ import (
 	"bytes"
 	"fmt"
 
-	"github.com/docker/go-units"
 	"github.com/tikv/pd/pkg/btree"
 	"github.com/tikv/pd/pkg/keyutil"
 	"github.com/tikv/pd/pkg/slice"
 	"github.com/tikv/pd/server/core"
 	"github.com/tikv/pd/server/statistics"
 )
 
-var minHotThresholds = [statistics.RegionStatCount]uint64{
-	statistics.RegionReadBytes:  8 * units.KiB,
-	statistics.RegionReadKeys:   128,
-	statistics.RegionReadQuery:  128,
-	statistics.RegionWriteBytes: 1 * units.KiB,
-	statistics.RegionWriteKeys:  32,
-	statistics.RegionWriteQuery: 32,
+var minHotThresholds [statistics.RegionStatCount]uint64
+
+func init() {
```

Comment:
Why do we need to make a copy in the init function?

Reply:
Their values are the same.